apps/fluent-tester/src/FluentTester/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { ACTIVITYINDICATOR_TESTPAGE } from './consts';
+import { View, StyleSheet, Animated, Easing, ViewProps } from 'react-native';
+
+const radius = 40; // from center to halfway border
+const border = 10; // line thickness
+const xydist = Math.sqrt(2)/2 * radius;
+
+// let R = big circle radius, from middle of circle to middle of border width = (width - borderRadius) / 2
+// borderRadius only refers to big circle radius
+// D = sqrt(2)/2 * R is distance from center to the middle of end circle
+// Top end circle:
+  // top: R - D - borderRadius
+    // top edge is already 1 borderRadius below the big circle
+  // left: R - D - borderRadius
+    // left edge is already 1 borderRadius to the right
+// Bottom end circle:
+  // top: R + D - 2 * borderRadius
+    // 1 borderRadius explained same as top end circle. Extra borderRadius since bottom end circle is 1 borderRadius below top end circle already.
+  // left: R - D - borderRadius
+    // same explanation as above
+
+const styles = StyleSheet.create({
+  semicircle: {
+    width: radius*2 + border, // from outside edge of circle on one side to the opposite side
+    height: radius*2 + border,
+    borderWidth: border,
+    borderColor: '#919191',
+    borderRadius: radius + border/2,
+    borderLeftColor: 'transparent',
+  },
+  endTop: {
+    width: border,
+    height: border,
+    borderRadius: border/2,
+    backgroundColor: '#919191',
+    top: radius - xydist - border,
+    left: radius - xydist - border,
+  },
+  endBottom: {
+    width: border,
+    height: border,
+    borderRadius: border/2,
+    backgroundColor: '#919191',
+    top: radius + xydist - 2*border,
+    left: radius - xydist - border,
+  }
+});
+
+
+export type ActivityIndicatorProps = ViewProps & {
+    animating?: boolean;
+    hidesWhenStopped?: boolean;
+}
+const ActivityIndicator = (props: ActivityIndicatorProps) => {
+  const spinAnimation = useRef(new Animated.Value(0)).current;
+  if (props.animating) {
+    React.useEffect(() => {
+      Animated.loop(
+          Animated.timing(spinAnimation, {
+            toValue: 359,
+            duration: 750,
+            useNativeDriver: true,
+            easing: Easing.linear
+          })
+      ).start();
+    });
+  }
+  const interpolateSpin = spinAnimation.interpolate({
+    inputRange: [0, 359],
+    outputRange: ['0deg', '359deg'],
+  });
+
+  // or display: none, but since React Native ActivityIndicator still takes up space when hidden, using opacity
```

Comment:
This may not be in scope, but one thing about using opacity to control hidden-ness is that it doesn't trick screen readers, so they will still announce the presence of things hidden in this way. If I remember correctly display:none works better for screen readers, but I might also just be making things up.

Reply:
If the opacity is only for animating and the component is removed when hidden, then this is fine as it is

Reply:
Also just realized this is tester code, so all of this is probably a moot point.

Reply:
This is all tester code... but it should actually be one of the props :P 

I'm actually in favor of ignoring this prop altogether. It feels like a legacy prop and clients can hide views themselves by modifying the style.. and also less code for us to maintain. It would introduce a difference between the Fluent ActivityIndicator and the core one, but I think that's fine...

Reply:
Thanks for the points @rurikoaraki , bringing up that just changing the opacity would not work well for accessibility